# HG changeset patch
# User Ben Abbott <bpabbott@mac.comg>
# Date 1450918164 18000
#      Wed Dec 23 19:49:24 2015 -0500
# Node ID 34d0497730e6f133ef430e782e354b3fb19bd0f1
# Parent  984678f19352d1f630122e6f03b1f6a404ff713d
Include missing "{}" for @LaTeX{} macro in print.m doc-string,
introduced in rev 123da81b9176.

* print.m: Include missing "{}" for @LaTeX{} macro.

diff --git a/scripts/plot/util/print.m b/scripts/plot/util/print.m
--- a/scripts/plot/util/print.m
+++ b/scripts/plot/util/print.m
@@ -164,17 +164,17 @@
 ##   @item hpgl
 ##     HP plotter language
 ##
 ##   @item  jpg
 ##   @itemx jpeg
 ##     JPEG image
 ##
 ##   @item latex
-##     @LaTeX picture environment (only available with the gnuplot graphics
+##     @LaTeX{} picture environment (only available for the Gnuplot graphics
 ## toolkit).
 ##
 ##   @item mf
 ##     Metafont
 ##
 ##   @item png
 ##     Portable network graphics
 ##
